ipatests/test_ipalib/test_text.py

```diff
@@ -201,6 +201,12 @@ def test_mod(self):
         inst = self.klass('hello %(adj)s nurse', 'foo', 'bar')
         assert inst % dict(adj='naughty', stuff='junk') == 'hello naughty nurse'
 
+    def test_format(self):
+        inst = self.klass('{0} {adj} nurse', 'foo', 'bar')
+        posargs = ('hello', 'bye')
+        knownargs = {'adj': 'naughty', 'stuff': 'junk'}
+        assert inst.format(*posargs, **knownargs) == 'hello naughty nurse'
```

Comment:
I realize you are sticking to previous convention but I think a new nurse adjective should be chosen with a more positive message. Perhaps "compassionate". I'll see about changing the others.

Reply:
FWIW I'm going with the adjective "tall" in my patch.

Reply:
'caring nurse' is positive i think.

Reply:
Sounds good to me!

Reply:
Yeah, better than what I came up with :-)